library/src/buildinfo.cpp

```diff
@@ -10,7 +10,7 @@
 #define TO_STR(x) TO_STR2(x)
 #define VERSION_STRING                                                           \
     (TO_STR(ROCBLAS_VERSION_MAJOR) "." TO_STR(ROCBLAS_VERSION_MINOR) "." TO_STR( \
-        ROCBLAS_VERSION_PATCH) "." TO_STR(ROCBLAS_VERSION_TWEAK) "-" TO_STR(ROCBLAS_VERSION_COMMIT_ID))
+        ROCBLAS_VERSION_PATCH) "." TO_STR(ROCBLAS_VERSION_TWEAK) )

```

Comment:
change
ROCBLAS_VERSION_PATCH) "." TO_STR(ROCBLAS_VERSION_TWEAK) )
to
ROCBLAS_VERSION_PATCH) "." TO_STR(ROCBLAS_VERSION_TWEAK))
in order to pass clang-format. There should not be space between ) ), it should be ))

Reply:
thanks, I should be checking the dozens of Jenkins emails I am getting.